packages/plugin-dialog/src/ui/DialogZoneNode.ts

```diff
@@ -0,0 +1,3 @@
+import { ZoneNode } from '../../../plugin-layout/src/ZoneNode';
+
+export class DialogZoneNode extends ZoneNode {}
```

Comment:
See the comment on ZoneNode.
I don't see why a DialogZone need to inherit from a ZoneNode. It could however have a flag that tells that it manages zones in order to receive "signals".